docs/getting-started/create-pipeline.md

```diff
@@ -0,0 +1,61 @@
+# Creating a Pipeline
+Using the Screwdriver CLI, you can build pipelines locally on your machine.
+
+## Set up Screwdriver CLI
+
+**The Screwdriver CLI does not exist yet, but here is what the instructions would looks like.**
+
+### Installing the CLI
+
+```bash
+$ go get github.com/screwdriver-cd/client
+$ $GOPATH/bin/client
+```
+
+### Configure your Git token
+Get the URL to login to Screwdriver
+
+```bash
+$ sd login  # prompts you to visit http://the-api-url.us-west-2.elb.amazonaws.com/v3/login
+```
+
+Copy your token and paste it
+```bash
+$ sd config token {YOUR_TOKEN_HERE}
+```
+
+
+## Using the Screwdriver CLI
+Now that the Screwdriver CLI is set up, you can use it to create a pipeline. Creating a pipeline will automatically create a `main` job and start a build.
+
+### Start the CLI
+To enter the read-eval-print loop, run:
+
+```bash
+$ sd repl
+```
+
+### Create a Pipeline
+To create a pipeline, you will need your `scmUrl`.
+```bash
+sd$ pipelines create {YOUR_SCMURL}  # (ex: git@github.com:screwdriver-cd/hashr.git#master)
+```
+Congratulations! You just ran your first build!
```

Comment:
I don't believe that the `pipeline create` actually runs the build. It creates the job though